### PR TITLE
update icons in GB sidebar toolstrip

### PR DESCRIPTION
**Pull request by ridwanamo**
## Description
Remove the following icons:
* TrackManager
* Personal Data

Add one icon: Navigate

Confirm with Andrea size of navigate icon should be 26px x 26px and tooltip on icon is "Navigate browser image"

Had to update one test.

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1771

## Deployment URL(s)
http://toolstrip-update.review.ensembl.org

## Views affected
Genome browser only

### CODE REVIEW

File: src/content/app/genome-browser/components/browser-sidebar-toolstrip/BrowserSidebarToolstrip.tsx
Comment (azangru) [inline src/content/app/genome-browser/components/browser-sidebar-toolstrip/BrowserSidebarToolstrip.tsx:125]: why so many arrays?

File: src/shared/components/layout/StandardAppLayout.scss
Comment (azangru) [inline src/shared/components/layout/StandardAppLayout.scss:136]: Wait, you still had all this padding (green) to shave off before starting tweaking width and height.

Screenshot 2022-11-04 at 15 12 57

BTW: might be worth waiting until the updated ImageButton component is merged; and check how it works there.

File: src/content/app/genome-browser/components/browser-sidebar-toolstrip/BrowserSidebarToolstrip.tsx
Comment (ridwanamo) [reply src/content/app/genome-browser/components/browser-sidebar-toolstrip/BrowserSidebarToolstrip.tsx:125]: sorry my bad, i was trying something else before and forgot to remove the square bracket. Done now.

File: src/shared/components/layout/StandardAppLayout.scss
Comment (ridwanamo) [reply src/shared/components/layout/StandardAppLayout.scss:136]: So the padding is set to 4px on all the icons....whats the better solution, set the padding to 2px or set the height and width? Feels like we might want to keep the padding for the icons the same isn't it? I

Comment (azangru) [reply src/shared/components/layout/StandardAppLayout.scss:137]: The updated `ImageButton` component has `--image-button-svg-width` and `--image-button-svg-height` properties. You can use either them or the padding to tweak the icon size; whichever is more convenient/appropriate.

Comment (ridwanamo) [reply src/shared/components/layout/StandardAppLayout.scss:137]: I went with the padding instead of readjusting the width and height because the padding was affecting the size of the image so that sounds like a better fix.

Comment (azangru) [reply src/shared/components/layout/StandardAppLayout.scss:137]: > so that sounds like a better fix

I'm confused by why you would consider this a better fix; and drew a diagram to explain what I meant:

image

I'm not saying that this is better or worse than the padding; I'm just saying that this allows you to control the dimensions of the svg directly, rather than calculating them from image button width(height) - padding *2.

**Review by azangru — APPROVED** [review]

File: src/shared/components/layout/StandardAppLayout.scss
Comment (ridwanamo) [reply src/shared/components/layout/StandardAppLayout.scss:137]: I am also confused. I understand your point of being able to override the image width and height using the css variable. So which one should we go with:
* set padding to 2px or
* set width and height to 24px

Comment (azangru) [reply src/shared/components/layout/StandardAppLayout.scss:137]: 24x24px is the dimensions of the ImageButton, which is this rectangle:



whereas the `--image-button-svg-width` and `--image-button-svg-height` properties control the size of the svg image inside the rectangle:



Both the padding and the CSS variable are valid options for setting the size of the svg image inside the button. The option with the variable is more straightforward because it is easier to reason about than the padding option (e.g. `--image-button-svg-width: 20px` and `--image-button-svg-height: 20px` gives you an svg with a size of 20x20px; whereas `padding: 4px` means that the size of the svg is the size of the ImageButton (24x24px) minus twice the size of the padding (-4px left, -4px right, -4px top, -4px bottom)).

File: src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx
Comment (imransl) [inline src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx:21]: What is the purpose of this CSS class?

File: src/styles/main.scss
Comment (ridwanamo) [reply src/styles/main.scss:30]: Removed it and it still works fine.

File: src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx
Comment (ridwanamo) [reply src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx]: I have removed the css class. Its funny though all the other modal has the same thing.

**Review by azangru — APPROVED** [review]

File: src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx
Comment (imransl) [reply src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx]: We need to remove them too if we aren't using them. But let's do that in another PR :)

**Review by imransl — APPROVED** [review]

**Pull request merged**
